### ISSUE TITLE
example spec file too long

### ISSUE THREAD

**Issue by bencao** [cypress-io/cypress-example-kitchensink#9]
The example_spec.js file is too long.
Could we cut it into separate files, each demonstrate a specific functionality like request_spec.js etc?

**Comment by brian-mann** [comment]
Agreed it is too long - but this is the file that is "seeded" when new projects are created.

We could either combine them back into one file just for the seeding, or seed multiple files (which may be annoying).

I think another option is during the seeding phase to give users a button that's like "I know what I'm doing, please remove these files..."

**Comment by bencao** [comment]
how about using an examples/ directory to hold them all?

**Comment by bahmutov** [comment]
Yeah, I am all for splitting this long spec file into many files showing different types of tests. And our scaffolding could generate multiple files as well

**Comment by brian-mann** [comment]
Definitely +1. We've talked internally about changing the entire scaffolding process.

Not only is it too long but running 90 tests in GUI mode is not the intended use case and has many problems on its own.

Splitting it up more accurately reflects how you should be using GUI mode, will be less intimidating, and help organize the various commands into their separate concerns.

**Comment by bahmutov** [comment]
Adding better `cypress init` command here https://github.com/cypress-io/cypress/issues/619

**Comment by brian-mann** [comment]
The GUI also needs to be updated to prompt the user, but allow them to decline scaffolding the example files.

Only power users using Cypress on a bunch of projects will use the `cypress init` command line. Whereas the vast majority of our users will go through the GUI.

**Comment by jennifer-shehane** [comment]
Related to: https://github.com/cypress-io/cypress/issues/681
